esiclient/tests/unit/v1/test_node_network.py

```diff
@@ -106,14 +112,32 @@ def test_take_action(self):
 
         parsed_args = self.check_parser(self.cmd, arglist, verifylist)
 
+        results = self.cmd.take_action(parsed_args)
+        expected = (
+          ["Node", "MAC Address", "Port", "Network", "Fixed IP"],
+          [['node1', 'aa:aa:aa:aa:aa:aa', 'node1', 'test_network', '1.1.1.1'],
+           ['node1', 'aa:aa:aa:aa:aa:aa', None, None, None],
+           ['node2', 'bb:bb:bb:bb:bb:bb', None, None, None],
+           ['node2', 'cc:cc:cc:cc:cc:cc', 'node2', 'test_network', '2.2.2.2'],
+           ['node2', 'cc:cc:cc:cc:cc:cc', None, None, None]]
+        )
```

Comment:
This is definitely not expected; now we're getting the same mac address returned multiple times, once with the network fields filled out and once without. That means there's something wrong with the updated code.